<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Focuser Server</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 31, 49);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 132, 228);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 28, 40);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="boxLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Show LOG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Minimizes program to tray.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
background-color: rgb(119, 118, 123);</string>
    </property>
    <property name="text">
     <string>Hide</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>6</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Focuser160MQ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="title">
     <string>Connectivity</string>
    </property>
    <widget class="QLabel" name="lblIP">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>30</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPort">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>60</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblCommSpeed">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>90</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Moving</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Error</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblMov">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblErr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPing">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Device (Un)Reachable</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>ClientId:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtClientID">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Current Client</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblEnc">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Encoder</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="statServer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="statRouter">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="statMotor">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Router</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);</string>
    </property>
    <property name="text">
     <string>Motor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lineRM">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>100</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: forestGreen;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lineSR">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>100</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: forestGreen;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>btnStart</zorder>
   <zorder>btnStop</zorder>
   <zorder>boxLog</zorder>
   <zorder>btnHide</zorder>
   <zorder>label</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lblPos</zorder>
   <zorder>lblMov</zorder>
   <zorder>lblErr</zorder>
   <zorder>lblPing</zorder>
   <zorder>label_5</zorder>
   <zorder>txtClientID</zorder>
   <zorder>lblEnc</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>lineRM</zorder>
   <zorder>statMotor</zorder>
   <zorder>lineSR</zorder>
   <zorder>statServer</zorder>
   <zorder>statRouter</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>259</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 221, 218);</string>
   </property>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuTests">
    <property name="title">
     <string>Tests</string>
    </property>
    <addaction name="actionClient_Simulator"/>
   </widget>
   <addaction name="menuSetup"/>
   <addaction name="menuTests"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionClient_Simulator">
   <property name="text">
    <string>Client Simulator</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
